.github/workflows/e2e.yml

```diff
@@ -0,0 +1,45 @@
+name: integration test
+
+on:
+  push:
+    branches:
+      - "main"
+    tags:
+      - v[0-9]+.[0-9]+.[0-9]+
+  pull_request:
+    branches:
+      - "main"
+
+env:
+  CARGO_TERM_COLOR: always
+  REGISTRY: ghcr.io
+
+jobs:
+  run-e2e:
```

Comment:
Is the job duplicated with [this](https://github.com/containerd/nydus-snapshotter/blob/main/.github/workflows/image.yml) ?

Reply:
Yes, you are right. In fact, most containerized e2e test cases are copied from it with replacing crictl with nerdctl. The stargz case is still not migrated. So let's keep them two run concurrently for a while until stargz case is added.

Reply:
Maybe we should remove it from this PR? otherwise nobody knows we need to remove the duplicated job in the future.

Reply:
It is deleted.